test-cases/longevity/longevity-change-cluster-size-by-2-times.yaml

```diff
@@ -0,0 +1,26 @@
+test_duration: 800
+
+prepare_write_cmd:  "cassandra-stress write cl=QUORUM n=20971520 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=80 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
+
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=720m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=40 -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
+             "cassandra-stress read  cl=QUORUM duration=720m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=40 -pop 'dist=uniform(1..20971520)' -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
+             ]
+
+n_db_nodes: 4
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+
+nemesis_class_name: 'GrowShrinkClusterNemesis'
+nemesis_interval: 5
+nemesis_filter_seeds: false
+nemesis_add_node_cnt: 4
+
+seeds_num: 3
+
+user_prefix: 'raft-double-cluster-size-with-schema-changes-12h'
```

Comment:
Why is it called "-with-schema-changes"?
Where are the schema changes?

Reply:
fixed with nemesis filter